src/Dto/TaskDto.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Dto;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class TaskDto
+{
+    public function __construct(
+        #[Assert\NotBlank()]
+        #[Assert\Positive()]
+        public readonly int $id,
```

Comment:
I suggest moving the `$id` to the controller as a request parameter to follow REST API principles